manim/utils/simple_functions.py

```diff
@@ -108,17 +125,20 @@ def get_parameters(function: Callable) -> MappingProxyType[str, inspect.Paramete
     --------
     ::
 
-        get_parameters(get_parameters)
-        # Returns mappingproxy(OrderedDict([('function', <Parameter "function: 'Callable'">)]))
+        >>> get_parameters(get_parameters)
+        mappingproxy(OrderedDict([('function', <Parameter "function: 'Callable'">)]))
 
-        dict(get_parameters(choose))
-        # Returns {'n': <Parameter "n: 'int'">, 'k': <Parameter "k: 'int'">}
+        >>> tuple(get_parameters(choose))
+        ('n', 'k')
     """
     return inspect.signature(function).parameters
 
 
 def sigmoid(x: float) -> float:
-    """Returns the output of the logistic function, defined as 1/(1+e^-x)
+    r"""Returns the output of the logistic function.
+    
+    The logistic function, or sigmoid function, is defined
```

Comment:
```suggestion
    The logistic function, a common example of a sigmoid function, is defined
```
A minor [technicality on terminology](https://en.wikipedia.org/wiki/Sigmoid_function#:~:text=A%20common%20example%20of%20a%20sigmoid%20function%20is%20the%20logistic%20function) if it matters.

Reply:
You are perfectly correct!